xilem/src/view/sized_box.rs

```diff
@@ -4,13 +4,22 @@
 use std::marker::PhantomData;
 
 use masonry::widget;
+use vello::kurbo::RoundedRectRadii;
+use vello::peniko::{Brush, Color};
 use xilem_core::ViewMarker;
 
 use crate::{
     core::{Mut, View, ViewId},
     Pod, ViewCtx, WidgetView,
 };
 
+/// Something that can be used as the border for a widget.
+#[derive(PartialEq)]
+struct BorderStyle {
+    width: f64,
+    color: Color,
+}
```

Comment:
Since this isn't public API, we should move this to below `SizedBox`

Reply:
Done.